src/main/kotlin/solve/settings/visualization/popover/LayerSettingsPopOverNode.kt

```diff
@@ -0,0 +1,27 @@
+package solve.settings.visualization.popover
+
+import javafx.scene.Node
+import javafx.scene.control.Label
+import javafx.scene.layout.HBox
+import javafx.scene.text.Font
+import solve.utils.createHGrowHBox
+import tornadofx.*
+
+abstract class LayerSettingsPopOverNode {
```

Comment:
Use composition instead of inheritance. For abstraction you can have interface for different PopOverNodes, with method getNode or something like that

Reply:
Still actual